static/index.html

```
@@ -319,6 +403,89 @@ <h4>Crop Image</h4>
+                            <option value="jpg">JPG</option>
+                            <option value="png">PNG</option>
+                            <option value="webp">WebP</option>
+                        </select>
+                        <label class="helper-text" for="convert-format-quality">Quality (jpg/webp): <span id="convert-format-quality-value">90</span></label>
+                        <input type="range" id="convert-format-quality" min="10" max="100" value="90">
+                        <button id="process-convert-format-btn" class="primary-btn">Convert</button>
+                    </div>
+
+                    <!-- Watermark image options -->
+                    <div id="watermark-image-area" class="password-area hidden">
+                        <input type="text" id="watermark-image-text" placeholder="Watermark text">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)